docs/framework/docker/aspnetmvc.md

```diff
@@ -0,0 +1,17 @@
+--
+title: Running ASP.NET MVC applications in Docker
+description: Running ASP.NET MVC applications in Docker
```

Comment:
Updated, except for the descriptions on the placeholders. Let's add those as we fill in the documents.